x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/index.tsx

```diff
@@ -187,6 +201,39 @@ export const CreatePackagePolicyPage: React.FunctionComponent = () => {
     [packageInfo, setAgentPolicy, setFormState]
   );
 
+  const setPolicyValidation = (
```

Comment:
it's a bit tricky since `setHasAgentPolicyError` is being passed down to child components too to bubble up errors, I could change to make the submit button `disabled` condition more complex